From: Antonio Terceiro <terceiro@debian.org>
Date: Wed, 2 Feb 2022 15:11:02 -0300
Subject: lxc-net: don't start by default inside lxc

When lxc is installed inside an lxc container, trying to bring up
lxc-net with the default parameters will conflict with the networking
setup for lxc on the host. This breaks all networking inside the
container where lxc is installed.

Signed-off-by: Antonio Terceiro <terceiro@debian.org>
Forwarded: https://github.com/lxc/lxc/pull/4077
---
 config/init/systemd/lxc-net.service.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/config/init/systemd/lxc-net.service.in b/config/init/systemd/lxc-net.service.in
index 6ed8cd8..a615d8b 100644
--- a/config/init/systemd/lxc-net.service.in
+++ b/config/init/systemd/lxc-net.service.in
@@ -3,6 +3,7 @@ Description=LXC network bridge setup
 After=network-online.target
 Before=lxc.service
 Documentation=man:lxc
+ConditionVirtualization=!lxc
 
 [Service]
 Type=oneshot
